src/olympia/stats/templates/stats/addon_report_menu.html

```diff
@@ -3,6 +3,7 @@
     <li data-report="overview" data-layout="overview">
       <a href="{{ url('stats.overview', addon.slug) }}">{{ _('Overview') }}</a>
     </li>
+    {% if addon.has_listed_versions() %}
```

Comment:
Do we show stats for old versions after they've been deleted?  if so we might want `include_deleted=True` here.

Reply:
> Do we show stats for old versions after they've been deleted?

Yeah, I think that's possible so let's included deleted versions, that probably does not hurt.